.changeset/fifty-garlics-heal.md

```diff
@@ -0,0 +1,5 @@
+---
+"@remix-run/dev": patch
+---
+
+fix behind-by-one-render bug for meta hmr
```

Comment:
Do we need to call out that it's specific to Vite?

Reply:
this changeset is just here to make sure a pre-release can get triggered. won't include this changeset in any changelogs